mlir/utils/jenkins/MIOpenDriver.py

```diff
@@ -192,6 +188,7 @@ def __init__(self, argv, xdlops):
 
         
 def runConfigWithMLIR(config):
+    os.system("rm "+BENCHMARKING_RESULT_FILE_NAME)
```

Comment:
Same applies for `runConfigWithMIOpen`:

Why don't we remove the benchmarking result after the benchmark has finished, rather than before it get started?

I think it can be more cleanly done in `generatePerformanceResults` where it can get rid of this file after benchmarking is done for both routines.

Reply:
Either way would work. Doing it before benchmarking can ensure the old result gets cleared no matter what happened in previous benchmarking. To do it after benchmarking, it needs to be done in generatePerformanceResults, benchmarkMIOpen, and benchmarkMLIR.

Reply:
I'd be good with you continue as-is but add a comment about the explanation you had in here. Also mention that rocprof (I assume) populated this file after the benchmarking.

Reply:
Much delayedly, how about https://docs.python.org/3/library/os.html#os.remove ?